sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -80,6 +80,20 @@ export class Sender {
     return sender.send(message);
   }
 
+  /**
+   * Ensures that the Service Bus connection and sender link is open.
+   *
+   * This method will be called on-demand by the Sender. Only call this method if you need
+   * to force the opening of the connection and link.
+   */
+  async open() {
+    // the underlying `MessageSender` initialization will both
+    // cache the sender in the context for the next send and
+    // ensure the link and connection are initialized.
+    const tempSender = MessageSender.create(this._context);
```

Comment:
Why call it the "temp" sender? Both the `send()` and the `sendBatch()` methods call it simply the "sender". Calling it any different here would imply that this is different

Reply:
I can rename it - in the other cases they create it and send() so the utility of why it's there is obvious.

In this one I create it, open() it and then throw it away. I wanted some signal to the reader that there's something interesting happening here they need to pay attention to.

Reply:
> In this one I create it, open() it and then throw it away

In the others, we create it, send() it and then throw it away

How about `return MessageSender.create(this._context).open()`?

Reply:
> In the others, we create it, **send()** it and then throw it away

The send() bit is the key difference. It's pretty common to create something, send (or read a message) and then throw it away.

It's really _uncommon_ to create something, open() it and then throw it away :)

Changed to be the one-liner just to have emotional closure. :)